common/app/views/fragments/stylesheets.scala.html

```diff
@@ -58,8 +58,8 @@
   - Exclude IE Mobile [&(!IEMobile)]
 *@
 <!--[if (IE 9)&(!IEMobile)]>
-    @fragments.stylesheetLink(Static.css.headIE9(projectName))
-    @fragments.stylesheetLink("stylesheets/ie9.content.css")
+    @fragments.stylesheetLink(Static.css.headIE9(projectName), isCrossword)
```

Comment:
Not a major deal but I guess you don't need to disable this on `IE9` or `lt IE 9` if we're not seeing the issue in those browsers and keep the performance advantage there?